DESY/VBF_M125_SM_tt.txt

```diff
@@ -2,6 +2,16 @@
 
 PATH:
 
+/afs/cern.ch/user/d/dvats/public/for_sync/VBFHToTauTau_M125_13TeV_powheg_pythia8_v2.root
+
+UPDATES wrt v1:
+
+TauSpinner weight added (branches: sm_htt125 (even) ,ps_htt125 (odd))
+
```

Comment:
Hi,
please store TauSpinner weights using standard branch names (tauspinnerH, tauspinnerA, tauspinnerMaxMix respectively for CP-even, CP-odd, and max-mix, i.e. alpha=pi/4) as otherwise automated comparison of ntuples by different groups is difficult.

Reply:
Hi,
I put new file in the directory with required name changes. Please tell if you find some other errors.

Reply:
OK, thanks. I am merging this, but please check an e-mail from me about selection you possibly apply.